optimum/habana/transformers/models/paligemma/modeling_paligemma.py

```diff
@@ -49,17 +48,13 @@ def forward(
         return_dict: Optional[bool] = None,
         logits_to_keep: Union[int, torch.Tensor] = 0,
         token_idx: Optional[torch.Tensor] = None,
-        **lm_kwargs,
+        **kwargs,
     ) -> Union[tuple, PaliGemmaCausalLMOutputWithPast]:
         """
         Inherits from PaliGemmaForConditionalGeneration::forward https://github.com/huggingface/transformers/blob/v4.45.1/src/transformers/models/paligemma/modeling_paligemma.py#L402
         The only differences are:
         - add new args token_idx
         """
-
-        if (input_ids is None) ^ (inputs_embeds is not None):
```

Comment:
Should this be removed?

Reply:
What I mean is that the upstream now does it in the class `PaliGemmaModel` and not in `PaliGemmaForConditionalGeneration` anymore. That's why I'm removing it from `GaudiPaliGemmaForConditionalGeneration`.